modin/core/dataframe/base/dataframe/dataframe.py

```diff
@@ -103,10 +107,13 @@ def map(
             The function to map across the dataframe.
         axis : int or modin.core.dataframe.base.utils.Axis, optional
             The axis to map over.
-        dtypes : str, optional
+        dtypes : pandas.Series or scalar type, optional
             The data types for the result. This is an optimization
             because there are functions that always result in a particular data
             type, and this allows us to avoid (re)computing it.
```

Comment:
```suggestion
            The data types for the result. This is an optimization
            because there are functions that always result in a particular data
            type, and this allows us to avoid (re)computing it.
            If the argument is a scalar type, then that type is assigned to each result column.
```